docs/source/configuration/open_edx.rst

```diff
@@ -134,7 +134,7 @@ In MITx Online:
 
 In Open edX (derived from instructions `here <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/tpa_integrate_open/tpa_oauth.html#additional-oauth2-providers-advanced>`_):
 
-* ``make lms-shell`` into the LMS container and ensure the following settings:
+* ``make lms-shell`` into the LMS container and ensure the following settings are configured in ``/edx/etc/lms.yml``.  Search for the setting names before appending anything new to the file.:
```

Comment:
A very similar thing is done in https://github.com/mitodl/mitxonline/pull/683/files#diff-a90f6fb11bd555c7c3cf903cafc0798bf2abc9e796c6dfce046f9ffba5fbbd0dR137. That might conflict with this. If you agree with that you can remove this change from here.

Reply:
Ah yes, I just saw that this morning.  Thanks for the catch.